e2e_tests/pytest.ini

```diff
@@ -3,3 +3,4 @@ markers =
     s3: a test is using AWS S3
     management: a test is using Management API
     integrations: a test is testing an integration
+    lightning: a test is supposed to run with Lightning EcoSystem CI
```

Comment:
But it can be run for non lightning ecosystem, right (`skip_if_on_lightning_ecosystem`)?